layouts/partials/dl-asset.html

```diff
@@ -0,0 +1,36 @@
+{{/*
+
+  A partial to download an asset from a CDN at build time and serve it from our
+  server. It outputs HTML attributes of the form:
+
+  {src/href}="{permalink}" integrity="{hash}"
+
+  Parameters:
+
+  * `src` or `href`: a string, the URL of the asset to download. The name of the
+    parameter will be reused as the attribute for the permalink.
+  * `integrity`: a boolean, whether to output the integrity hash.
```

Comment:
```suggestion
  * `integrity`: a boolean, whether to output the integrity attribute. If unspecified, defaults to `false`.
```

Reply:
The parameter has been removed, so this does not apply anymore.